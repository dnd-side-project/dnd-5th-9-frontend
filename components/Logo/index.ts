export { default as Spacing } from "./Spacing";
export { default as Symbol } from "./Symbol";
export { default as Wordmark } from "./Wordmark";
