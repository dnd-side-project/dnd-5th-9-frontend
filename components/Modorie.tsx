import React from "react";

const Modorie = () => {
  return <div>도리도리 모도리</div>;
};

export default Modorie;
