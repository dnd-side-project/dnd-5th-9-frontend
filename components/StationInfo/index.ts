export { default as StationInfo } from "./StationInfo";
