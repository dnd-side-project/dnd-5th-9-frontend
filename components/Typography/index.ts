import { TypoStyle } from "./Typography.styled";

export { default as Typography } from "./Typography";
export { TypoStyle };
