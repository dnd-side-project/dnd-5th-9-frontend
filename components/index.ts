export * from "./Icon";
export * from "./Input";
export * from "./StationInfo";
export * from "./Typography";
