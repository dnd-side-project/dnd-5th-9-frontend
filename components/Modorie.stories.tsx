import React from "react";

import Modorie from "./";

export default {
  title: "Components/Modorie",
  component: Modorie,
};

export const Standard = () => <Modorie />;
