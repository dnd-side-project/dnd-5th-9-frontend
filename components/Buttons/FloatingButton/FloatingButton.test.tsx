import { render } from "@testing-library/react";
import { FloatingButton } from "@/components/Buttons";

it("matches snapshot", () => {
  const utils = render(<FloatingButton />);
  expect(utils.container).toMatchSnapshot();
});
