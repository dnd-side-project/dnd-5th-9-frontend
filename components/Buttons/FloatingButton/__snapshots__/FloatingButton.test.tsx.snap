// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="sc-cxpSdN gsFmek"
    color="black"
  >
    <svg
      class="sc-egiyK jrqYGl"
      color="black"
      fill="none"
      height="23"
      viewBox="0 0 30 30"
      width="23"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M21.2035 13.4418L11.1453 23.5H6.503L6.5 18.8547L16.5582 8.79651L21.2035 13.4418Z"
        fill-rule="evenodd"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.8"
        vector-effect="non-scaling-stroke"
      />
      <path
        d="M16.5582 8.79655L18.2903 7.06447C19.0429 6.31184 20.2632 6.31184 21.0158 7.06447L22.9355 8.98421C23.6882 9.73684 23.6882 10.9571 22.9355 11.7097L21.2034 13.4418V13.4418"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.8"
        vector-effect="non-scaling-stroke"
      />
    </svg>
  </div>
</div>
`;
